bootstrap/bootstrap.go

```diff
@@ -1001,7 +1001,11 @@ func (b *Bootstrap) updateGitMirror() (string, error) {
 	// If we don't have a mirror, we need to clone it
 	if !utils.FileExists(mirrorDir) {
 		b.shell.Commentf("Cloning a mirror of the repository to %q", mirrorDir)
-		if err := gitClone(b.shell, b.GitCloneMirrorFlags, b.Repository, mirrorDir); err != nil {
+
+		gitCloneMirrorFlags := b.GitCloneMirrorFlags
+		gitCloneMirrorFlags += "--mirror"
```

Comment:
If `b.GitCloneMirrorFlags` is `"-v"` will this result in `"-v--mirror"` without a space?
It's a shame we're passing this around as a string, but that would be a bigger refactor.
We could `shellwords.Split()`, `append`, and then `strings.Join()` it here, which seems pretty clunky but will give the correct result.

Or, just…

```suggestion
		gitCloneMirrorFlags += " --mirror"
```

Reply:
I've pushed a commit which addresses this.
(First time I've ever pushed a commit to a third-party PR branch like that 🎉)